posneg.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
```

Comment:
Great code - the only suggestion I'd have is for you to have had your program prompt the user for input. Other than that, your code compiled and ran appropriately! 👍